docker/development/Dockerfile

```diff
@@ -10,7 +10,9 @@ ENTRYPOINT ["/usr/src/app/entrypoint.sh"]
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
     curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
     curl -sS https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -cs`-pgdg main" | tee  /etc/apt/sources.list.d/pgdg.list && \
-    apt-get update && apt-get install -y nodejs yarn postgresql-client-12 ffmpeg imagemagick pdftk ghostscript sqlite3 --no-install-recommends && rm -rf /var/lib/apt/lists/* && apt-get clean
+    sed -i "s/deb\.debian\.org/ftp.halifax\.rwth-aachen.de/g" /etc/apt/sources.list && \
```

Comment:
Why this?

Reply:
I noticed downloadspeeds are kinda suboptimal and added a german mirror ...
I hope there is a better way to do this but I dont know of any, maybe debian has a progam like arch's [`reflector`](https://wiki.archlinux.org/index.php/reflector)?